grabbit/src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessor.groovy

```diff
@@ -24,12 +24,15 @@ import com.twcable.grabbit.proto.NodeProtos.Properties
 import com.twcable.grabbit.proto.NodeProtos.Property
 import groovy.transform.CompileStatic
```

Comment:
I believe defined interactions in a setup/given are effectively ignored.  Try moving to when: block